data/hai/unfettered side missions.txt

```diff
@@ -239,24 +201,27 @@ mission "Hai Legacy 1"
 			`	The ship then hits your weapon, disarming you with surgical accuracy using its Bullfrog Anti-Missiles. One of the two Hai comes, seemingly disappointed. "It looks like you angered the piloting AI enough to make it override the silence protocols we had put in place. You're lucky it did not kill you, but even so you are of no use to us if you think it wise to use your firearm on a spaceship. Farewell." They turn and leave without another word, and the ship flies off.`
 				decline
 			label lucky??
-			`	You manage to land a lucky hit on what you assume to be the ship's navigation system, because it comes down, seemingly crashing.`
+			`	You manage to land a lucky hit on what you assume to be the ship's navigation system, because it comes down, seemingly crashing. You feel like you hear a robotic voice but it is hard to tell with all of the noise.`
```

Comment:
```suggestion
			`	You manage to land a lucky hit on what you assume to be the ship's navigation system, because it comes down, seemingly crashing. You feel like you hear a robotic voice cry out, but it's hard to tell with all the noise.`
```